### ISSUE TITLE
jStat.poisson - error for a large lambda

### ISSUE THREAD

**Issue by orenbh1** [jstat/jstat#293]
jStat.poisson.cdf( 3000, 4000 ) = NaN
jStat.poisson.pdf( 3000, 4000 ) = NaN

Potential "less accurate" solution, using the CDF Normal approximation (mean=lambda, variance=lambda)
In this case you may calculate PDF as CDF(x)-CDF(x-1)
(actually for discrete it is PMF)

It looks like in  R they used Tylor series, I assumes it is more accurate.

When using the normal approximation
PDF( 3990, 4000 ) = CDF(3990,4000)-CDF(3989,4000)= 0.437184 - 0.430962. = 0.006222

R result:
> dpois(x=3990, lambda=4000) [1] 0.00623708

**Comment by kgryte** [comment]
@orenbh1 If all you need is the CDF and PMF of the Poisson distribution, you could consider using

- [`@stdlib/stats-base-dists-poisson-cdf`](https://github.com/stdlib-js/stats-base-dists-poisson-cdf)
- [`@stdlib/stats-base-dists-poisson-pmf`](https://github.com/stdlib-js/stats-base-dists-poisson-pmf)

For the values you describe in the OP, those packages return results matching R:

```
In [1]: alias2standalone('base.dists.poisson.cdf')
Out[1]: '@stdlib/stats-base-dists-poisson-cdf'

In [2]: alias2standalone('base.dists.poisson.pmf')
Out[2]: '@stdlib/stats-base-dists-poisson-pmf'

In [3]: base.dists.poisson.cdf(3990,4000) - base.dists.poisson.cdf(3989,4000)
Out[3]: 0.006237080320835864
```

An eventual goal is to have `jStat` eventually build on and leverage `stdlib` functionality, but we haven't had the time/resources to finish this effort. In the meantime, if there are gaps in `stdlib`, would be good to resolve first there, so that we don't duplicate effort in the future.

**Comment by orenbh1** [comment]
Thanks Athan, this looks good.
I currently not using Node, is there a CDN link to script that I can add to the HTML to use the functions?
<script src="" >

**Comment by kgryte** [comment]
@orenbh1 Yes, see the installation instructions which link to builds for loading directly via a `script` tag: https://github.com/stdlib-js/stats-base-dists-poisson-cdf#installation. You can use either the `esm` or the `umd` branch builds.

**Comment by kgryte** [comment]
E.g., to use the UMD bundle, see https://github.com/stdlib-js/stats-base-dists-poisson-pmf/blob/umd/README.md

```html
<script type="text/javascript" src="https://cdn.jsdelivr.net/gh/stdlib-js/stats-base-dists-poisson-pmf@umd/browser.js"></script>
```

**Comment by orenbh1** [comment]
Thanks Athan :), I already tried it, but I was looking for poisson.cdf instead of cdf ... 
Now it is working :)

Looking at the "stdlib" readme file I can see they link separately for each function, is there a one link to the entire library?

**Comment by kgryte** [comment]
Depends on how much you want to use in order to determine whether that is a good idea. If you just want to use a couple of functions, linking to each separately is probably better. Otherwise, you can load namespaces.

For example, to load all Poisson APIs, see [`@stdlib/stats-base-dists-poisson`](https://github.com/stdlib-js/stats-base-dists-poisson).

To load all "base" distributions, see [`@stdlib/stats-base-dists`](https://github.com/stdlib-js/stats-base-dists).

..and so on and so forth.

**Comment by orenbh1** [comment]
Can you load the entire stdlib? like jStat (or is it bigger)
Is there any  one doc that contains all the functions of stdlib with description?

**Comment by kgryte** [comment]
If you are not using Node.js and just wanting to load directly in a browser, you can load top-level namespaces (e.g., [`@stdlib/stats`](https://github.com/stdlib-js/stats)). You likely don't want or need the entirety of `stdlib` (it's many times bigger than jStat, and includes functionality outside of statistics). Also, at this point, you are better off asking questions in official stdlib comms channels, rather than on the jStat issue tracker.
